README.md

```diff
@@ -847,6 +857,14 @@ Flags:
 - :whale: `-q, --quiet`: Only display snapshots IDs
 - :whale: `--format`: Format the output using the given Go template, e.g, `{{json .}}`
 
+### :whale: nerdctl image prune
+Remove all unused images. Roughly equals with `docker image prune -a`.
+
+Usage: `nerdctl image prune [OPTIONS]`
+
+Flags:
+- :nerd_face: `-f, --force`: Ignore removal errors.
```

Comment:
`:whale:` for Docker-compatible options.

Also please document  unimplemented flags. (Please see other command sections for examples)

Reply:
I think I mentioned this in the PR, the `-f`  in Docker it is used to a "Do not prompt for confirmation", but in nerdctl it is used for ignoring error messages, I used the :nerd_face: to differentiate this.

Reply:
Unimplemented flags added for both command.

Reply:
Why not implement Docker style -f ?

Reply:
Since as far as I know, nerdctl doesn't really pop confirmation messages, also in every other command the `force` is used to ignore errors.

For my oppinion it's more important to maintain flag/feature parity within nerdctl first, then with docker, but if you insist, I can create a prompt and use the flag as it suposed to be used in docker, for both commands.

Reply:
I'd prefer emulating Docker's behavior